### PR TITLE
Force helm metacontroller scenario to wait for LoadBalancer CN-272

### PR DESCRIPTION
**Pull request by hakanmemisoglu**
* Add initContainer to wait for LoadBalancer to get its external IP
* Update Metacontroller version to 2.1.1

### CODE REVIEW

File: doc/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -353,31 +353,28 @@ To install Service-Per-Pod Decorator Controller, you need to execute the followi
...
-helm install hz-hazelcast \
-     --set service.type=LoadBalancer \
-     --set annotations.service-per-pod-label=statefulset.kubernetes.io/pod-name,annotations.service-per-pod-ports="5701:5701" \
-     hazelcast/hazelcast
+helm install hz-hazelcast -f service-per-pod/values.yaml hazelcast/hazelcast
 ----
 --
 ====
 
-[NOTE]
```
Comment (hasancelik) [inline doc/modules/ROOT/pages/index.adoc]: Why did you remove this Note?

**Review by hasancelik — CHANGES_REQUESTED** [review]
Could you update the Cleanup section as well? 

https://github.com/hazelcast-guides/kubernetes-external-client/blob/53e2b4d68b97e8b4f083b4167aa54df109f87395/doc/modules/ROOT/pages/index.adoc#clean-up-1

File: doc/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (hakanmemisoglu) [reply doc/modules/ROOT/pages/index.adoc]: It is not needed anymore, right? It fixes the situation mentioned in the Note section.

Comment (hakanmemisoglu) [reply doc/modules/ROOT/pages/index.adoc]: Or do you want me to keep it for kubectl section?

Comment (hasancelik) [reply doc/modules/ROOT/pages/index.adoc]: > Or do you want me to keep it for kubectl section?

`Note` section is not `Helm and Metacontroller` tab specific so we should keep it. I think we should provide more detailed version of it maybe?

File: service-per-pod/values.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+service:
+  type: "LoadBalancer"
+annotations:
+  service-per-pod-label: "statefulset.kubernetes.io/pod-name"
+  service-per-pod-ports: "5701:5701"
+initContainers:
+  - name: wait-for-lb
+    image: eu.gcr.io/hazelcast-33/kubectl-jq:0.1.0
```
Comment (SeriyBg) [inline service-per-pod/values.yaml]: I guess this image won't be available to the public, right? Wouldn't it be better to have some image that can be fetched from `docker-hub` instead?

**Review by SeriyBg — COMMENTED** [review]
Just one small question

File: service-per-pod/values.yaml (hunk 2 above)
Comment (hakanmemisoglu) [reply service-per-pod/values.yaml]: Yes, we will change it once the public image is available. Actually, we even do not need `jq` anymore, any image with `kubectl` will be sufficient.

**Review by SeriyBg — APPROVED** [review]
LGTM

**Review by hasancelik — APPROVED** [review]
